app/models/state_changer/standard_plate.rb

```diff
@@ -56,8 +56,20 @@ def update_associated_requests
       raise_request_error if associated_requests.empty? && associated_submission?
 
       associated_requests.each do |request|
+        # customer_accepts_responsibility = nil
```

Comment:
Is this comment meant to be here? In what scenario is it nil?

Reply:
no, removed it
I think it's value was nil when I was running tests and was leaving myself a comment then forgot to remove it